scripts/mas_sso_config.sh

```diff
@@ -15,23 +13,23 @@ CONNECTOR_OPERATOR_ROLE=connector_fleetshard_operator
 ERR_MESSAGE="Keycloak server not running at localhost:8180. No realm configuration will be applied"
 timeout 120 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8180/auth/realms/master)" != "200" ]]; do echo "Waiting for keycloak server at localhost:8180"; sleep 10; done' || echo $ERR_MESSAGE
 
-RESULT=`curl -sk --data "grant_type=password&client_id=$CLIENT_ID&username=$USERNAME&password=$PASS" $KEYCLOAK_URL$TOKEN_PATH`
+RESULT=`curl -sk --data "grant_type=password&client_id=$CLIENT_ID&username=$KEYCLOAK_USER&password=$KEYCLOAK_PASSWORD" $URL$TOKEN_PATH`
```

Comment:
The `$(...)` syntax should be preferred to the old backticks syntax (see [this](http://mywiki.wooledge.org/BashFAQ/082) and [this](https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03))
```suggestion
RESULT=$(curl -sk --data "grant_type=password&client_id=$CLIENT_ID&username=$KEYCLOAK_USER&password=$KEYCLOAK_PASSWORD" $URL$TOKEN_PATH)
```

Reply:
@ziccardi - I saw that the whole script uses old syntax. If you have shellcheck plugin installed, it will complain about many syntax issues. do you think this should be addressed as a part of this PR?

Reply:
hmm.. probably would worth creating an issue about fixing all those issues then.